pkg/proxy/benchmarking/benchmark.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+FILE=${0}
+RED="\033[91m"
+BLUE="\033[94m"
+ENDCOLOR="\033[0m"
```

Comment:
Do we actually need colored output? This can be a pain in CI and we often wind up having to strip it out anyhow.

Consider using `ERROR` `INFO` etc instead

Reply:
I might be wrong here - 
This would not be used in CI. This is for local development only, setting a baseline benchmark and then trying out different optimizations and comparing the results

Reply:
That might be true today but we have other CI running performance testing and should probably leave the option readily available for the future